docs/ops.html

```diff
@@ -676,7 +676,7 @@ <h3><a id="java" href="#java">6.4 Java Version</a></h3>
 
   From a security perspective, we recommend you use the latest released version of JDK 1.8 as older freely available versions have disclosed security vulnerabilities.
 
-  LinkedIn is currently running JDK 1.8 u5 (looking to upgrade to a newer version) with the G1 collector. LinkedIn's tuning looks like this:
+  At the time when we write this, LinkedIn is running JDK 1.8 u5 (looking to upgrade to a newer version) with the G1 collector. LinkedIn's tuning looks like this:
```

Comment:
```suggestion
  At the time this is written, LinkedIn is running JDK 1.8 u5 (looking to upgrade to a newer version) with the G1 collector. LinkedIn's tuning looks like this:
```

Reply:
Good suggestion. Fixed in the commit https://github.com/apache/kafka/pull/8581/commits/355f93c0ac8dbc0723ea46c4780df2cbdc423c71 . Thank you